packages/cbioportal-utils/src/civic/CivicUtils.ts

```diff
@@ -88,67 +53,58 @@ export function getCivicVariants(
     mutationSpecs?: Array<MutationSpec>
 ): Promise<ICivicVariantIndex> {
     let civicVariants: ICivicVariantIndex = {};
-    let promises: Array<Promise<void>> = [];
-
     if (mutationSpecs) {
-        let calledVariants: Set<number> = new Set([]);
-        for (let mutation of mutationSpecs) {
-            let geneSymbol = mutation.gene.hugoGeneSymbol;
-            let geneEntry = civicGenes[geneSymbol];
-            let proteinChanges = [mutation.proteinChange];
+        const geneToProteinChangeSet: {
```

Comment:
@leexgh just took another look at this code and i think it could use a few unit tests.  it doesn't seem like the split protein changes are covered by the current spec, but i could be wrong.

Reply:
Add unit tests for this